### ISSUE TITLE
Missing comment strings in Translational

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1776]
**Reported by beutlich on 8 Sep 2015 15:03 UTC**
The following variables miss a descriptive comment string
```
Modelica.Mechanics.Translational.Interfaces.PartialFriction.mode
Modelica.Mechanics.Translational.Components.SupportFriction.s
Modelica.Mechanics.Translational.Components.Brake.s
Modelica.Mechanics.Translational.Components.MassWithStopAndFriction.stopped
Modelica.Mechanics.Translational.Components.MassWithStopAndFriction.PartialFrictionWithStop.mode
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1776

**Comment by modelica-trac-importer** [comment]
**Modified by ahaumer on 27 Sep 2015 16:58 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 13 Dec 2015 17:32 UTC**
Fixed in 66926c8fe0d2d903134964ae82f6ef2bff1f56fb

**Comment by modelica-trac-importer** [comment]
**Changelog modified by otter on 13 Dec 2015 17:32 UTC**
Added missing description strings in some components of the Translational library
